src/beans/Course.ts

```diff
@@ -20,7 +62,7 @@ class Course {
     const onlyLabs = this.sections.filter(
       (section) => isLab(section) && !isLecture(section)
     );
-    this.hasLab = onlyLectures.length && onlyLabs.length;
+    this.hasLab = !!onlyLectures.length && !!onlyLabs.length;
```

Comment:
This is a substantive change (type is changed from truthy/falsy number to proper boolean), but the only uses of `hasLab` are in conditional statements throughout `beans/` so it should be safe.